packages/web-react/src/components/Modal/demo/ModalHiddenCloseButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { Button } from '../../Button';
+import Modal from '../Modal';
+import ModalBody from '../ModalBody';
+import ModalDialog from '../ModalDialog';
+import ModalFooter from '../ModalFooter';
+import ModalHeader from '../ModalHeader';
+
+const ModalHiddenCloseButton = () => {
+  const [isOpen, setOpen] = useState(false);
```

Comment:
Btw. we have `useToggle` hook with the same functionality as these three lines ;-)

Reply:
changed